samples/akka-quickstart-java/README.md

```diff
@@ -0,0 +1,34 @@
+# Akka Quickstart with Java
+
+## Quickstart guide
+
+This example is described in https://doc.akka.io/docs/akka/current/typed/actors.html#first-example
+
+The Hello World example illustrates Akka basics. Within 10 minutes, you should be able to download and run the example and read the Introduction to Actors documentation to understand how the example is constructed. This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
+
+## How to run
+
+This template also provides build descriptors for maven and gradle. You can use any of the following commands to run the application:
+```
+mvn compile exec:exec
+```
+or
+```
+./gradlew run
```

Comment:
do we want to include gradle or not?

Reply:
For: we already have it, against: we don't really maintain the gradle/keep it up to date